src/main/java/at/hannibal2/skyhanni/features/misc/CollectionTracker.kt

```diff
@@ -130,10 +141,28 @@ object CollectionTracker {
         this.internalName = internalName
         itemName = name
         itemAmount = foundAmount
+        isDungeonBoss = false
+        dungeonFloor = null
 
         lastAmountInInventory = countCurrentlyInInventory()
+        ChatUtils.chat("Started tracking $itemName §ecollection.")
         updateDisplay()
+    }
+
+    private fun setNewDungeonBossCollection(floor: DungeonFloor) {
+        val foundAmount = ProfileStorageData.profileSpecific?.dungeons?.bosses[floor]
```

Comment:
Only safe (?.) or non-null asserted (!!.) calls are allowed on a nullable receiver of type MutableMap<DungeonFloor, Int>?